profile/CSIP.xml

```diff
@@ -808,10 +808,13 @@
                     </dl>
                 </description>
             </requirement>
-            <requirement ID="CSIP62" REQLEVEL="SHOULD" RELATEDMAT="VocabularyContentInformationTypeSpecification" EXAMPLES="fileSecExample1 fileSecExample2">
+            <requirement ID="CSIP62" REQLEVEL="SHOULD" RELATEDMAT="VocabularyContentInformationTypeSpecification CSIP2 CSIP114" EXAMPLES="fileSecExample1 fileSecExample2">
                 <description>
                     <head>Specific content type</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">An added attribute which describes the specific content information type specification used for the transferred content. The attribute is mandatory when the file group catagorisation value is "Representations". The vocabulary will evolve under the curation of the DILCIS Board as additional content information type specifications are developed.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">An added attribute which describes the specific content information type specification used for the transferred content.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The vocabulary will evolve under the curation of the DILCIS Board as additional content information type specifications are developed.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The attribute is mandatory when the file group catagorisation value is "Representations".</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">When the "Package type" is described with the value "Mixed" and or the file group describes a "Representation" this element describes the content type transferred in the file group.</p>
```

Comment:
When the "Package type" value is "Mixed" and/or the file group describes a "Representation", then this element describes the content type transferred in the file group.